conda-forge.yml

```
@@ -1,7 +1,12 @@
-build_platform: {osx_arm64: osx_64}
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
+  osx_arm64: osx_64
```

This was turned into a cross-compile build when it wasn't before. Reverting this...

```suggestion
build_platform:
  osx_arm64: osx_64
```

---

cc @conda-forge/core (as this is odd)

---

From offline discussion, appears to be due to this mini-migrator ( https://github.com/regro/cf-scripts/pull/1466 )